### ISSUE TITLE
Chrome Not Rendering Progress With Testem 1.16.1 and Jasmine 2.6.x

### ISSUE THREAD

**Issue by calebeno** [testem/testem#1127]
I have a custom testem setup using a locally installed version of Jasmine (testem is also run locally via an npm script).  This allows us to update Jasmine with new features as they come out and have testem automatically pick up those new features.  (In this case, new object comparison features when using the toEqual comparator are extremely useful.)  However, when I make the change to use Jasmine 2.6.x, Chrome fails to render the progress of the tests.  The page remains white with the only noticeable change being a progress indicator in the tab name showing a "run/total" count.  After the tests complete, the finished testem page finally renders.  There also seems to be associated memory issues as well.  The testem tab in the console is frequently unresponsive and occasionally must be forcibly closed.

Here is the setup:
package.json:
```javascript
"devDependencies": {
    "jasmine-core": "2.6.1",
    "testem": "1.16.1"
}
```

testem.json:
```javascript
{
    "framework": "custom",
    "test_page": "testem.mustache",
    "src_files": [
        ... app and test files ...
    ]
}
```

testem.mustache
```mustache
<!doctype html>
<html>
<head>
    <title>Testem</title>
    <link rel="stylesheet" href="node_modules/jasmine-core/lib/jasmine-core/jasmine.css">
    <script src="node_modules/jasmine-core/lib/jasmine-core/jasmine.js"></script>
    <script src="node_modules/jasmine-core/lib/jasmine-core/jasmine-html.js"></script>
    <script src="node_modules/jasmine-core/lib/jasmine-core/boot.js"></script>
    <script src="/testem.js"></script>
    {{#serve_files}}
    <script src="{{src}}"></script>
    {{/serve_files}}
</head>
<body>
<body>
    <div id="jasmine_content"></div>
</body>
</body>
</html>
```

Testem has been a great tool and I would love to see this issue fixed.  Thank you for your time looking into this.

**Comment by calebeno** [comment]
This seems to be resolved with Testem 1.17.0 and Jasmine 2.6.4

**Comment by johanneswuerbach** [comment]
Sorry for the delay and thanks for the update 👍
